src/lib/kit/components/Inputs/DateInput/DateInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useCallback} from 'react';
+
+import {DatePicker, DatePickerProps} from '@gravity-ui/date-components';
+import {StringInputProps} from '../../../../core';
+import {DateTime, dateTimeParse} from '@gravity-ui/date-utils';
+
+export interface DateProps
+    extends Omit<DatePickerProps, 'value' | 'disabled' | 'placeholder' | 'qa'> {}
+
+export const DateInput: React.FC<StringInputProps<DateProps>> = ({
+    name,
+    input,
+    spec,
+    inputProps,
+}) => {
+    const {value, onChange, onBlur, onFocus} = input;
+
+    const onUpdate = useCallback((date: DateTime | null) => {
+        if (!date) {
+            onChange('');
+        } else if (spec.viewSpec.dateInput?.outputFormat) {
```

Comment:
The data output format does not change

Uploading Screen Recording 2024-06-28 at 09.43.43.mov…